src/public/js/video-tagging/video-tagging.html

```diff
@@ -409,7 +417,7 @@
 
     getRegions: function(frameId) {
         let regions = null;
-        if (frameId >=0) {
+        if (frameId || frameId == 0) {
```

Comment:
In this case, any defined id should be valid since the frameId is used find a frame in the frames object which can have any arbitrary name based on the image file name or video frame number.